src/lints/new_struct_added.ron

```diff
@@ -0,0 +1,43 @@
+SemverQuery(
+    id: "new_struct_added",
+    human_readable_name: "new struct added",
+    description: "A new public struct has been exposed.",
+    required_update: Minor,
```

Comment:
```suggestion
    required_update: Minor,
    lint_level: Allow,
```
or
```suggestion
    required_update: Minor,
    lint_level: Warn,
```

With the new version of `cargo-semver-checks`, it is now possible to have allow-by-default and warn-by-default checks.

Reply:
I'd probably make this `Allow` by default. I'm sure many users will want to opt-in, but it might be a bit noisy to enable for everyone.